src/HelicityAngles.cxx

```diff
@@ -9,109 +9,70 @@
 #include "ParticleCombination.h"
 #include "StatusManager.h"
 
-namespace yap {
+#include "logging.h"
 
-//-------------------------
-HelicityAngles::HelicityAngles(Model& m) :
-    StaticDataAccessor(m, equal_up_and_down),
-    Phi_(RealCachedValue::create(*this)),
-    Theta_(RealCachedValue::create(*this))
-{
-    registerWithModel();
-}
+#include <assert.h>
```

Comment:
You don't need `logging.h` or `assert.h`.
I think you can also remove `CachedValue.h` and `CalculationStatus.h`